### ISSUE TITLE
Implement a substitute for GitHub downloads

### ISSUE THREAD

**Issue by rowanj** [rowanj/gitx#129]
Unfortunately, the GitHub peeps have decided to deprecate the project downloads support; so I'll need to implement something else.

**Comment by dentarg** [comment]
It would be great if http://rowanj.github.com/gitx/ had a link to the most recent version of GitX-dev. Found a link in an issue, so I'm good for the moment.

Btw, thanks for your work on GitX!

**Comment by rowanj** [comment]
Indeed, the project website needs to be updated similarly to the GitHub page.

**Comment by jasonnoble** [comment]
Is the link in the readme not the right link?  https://github.com/rowanj/gitx#download-the-latest-binary links to http://builds.phere.net/GitX/development/GitX-dev.dmg
